pkg/restic/common.go

```diff
@@ -39,25 +39,15 @@ const (
 	InitContainer               = "restic-wait"
 	DefaultMaintenanceFrequency = 24 * time.Hour
 
-	podAnnotationPrefix       = "snapshot.velero.io/"
+	// Deprecated.
+	podAnnotationPrefix = "snapshot.velero.io/"
+
 	volumesToBackupAnnotation = "backup.velero.io/backup-volumes"
 )
 
-// PodHasSnapshotAnnotation returns true if the object has an annotation
```

Comment:
I think you'll still need this function to maintain backwards-compatibility for restoring pre-v1.1 backups

Reply:
I'm thinking we should add an annotation so this doesn't seem to be completely unused code. Not remembering we have a standard. Going to add something.

Reply:
It's what we use now, so it has use for existing backups. The function should stick around for now to support them, as Steve says.

Are you thinking we continue to add annotations to PodVolumeBackups going forward, too?

Reply:
I don't think we use it now: https://github.com/heptio/velero/search?q=PodHasSnapshotAnnotation&unscoped_q=PodHasSnapshotAnnotation

Reply:
Sorry. I said annotation but I meant documentation to this function if it needs to stick around. I understand we use annotation still, but I'm saying this function in specific is not being used in the current version of the code.

Reply:
@skriss ^

Reply:
Oh - guess this is already not used! Delete away. Looks like our restore code uses the `GetPodSnapshotAnnotations` function instead.